doc-content/enterprise-only/vehicle-route-planning/run-vrp-manually-proc.adoc

```diff
@@ -74,12 +75,13 @@ In the following example, OptaWeb Vehicle Routing downloads the OSM map of Centr
 +
 [source,subs="attributes+"]
 ----
-java -jar optaweb-vehicle-routing-standalone-{MAVEN_ARTIFACT_VERSION}.jar \
---app.persistence.h2-dir=/home/user/.optaweb-vehicle-routing/db \
---app.routing.osm-dir=/home/user/.optaweb-vehicle-routing/openstreetmap \
---app.routing.gh-dir=/home/user/.optaweb-vehicle-routing/graphhopper \
---app.routing.osm-file=central-america-latest.osm.pbf \
---app.region.country-codes=BZ,GT
+java \
+-Dapp.demo.data-set-dir=$HOME/{VRP-DATA-DIR}/dataset \
+-Dapp.persistence.h2-dir=$HOME/{VRP-DATA-DIR}/db \
+-Dapp.routing.gh-dir=$HOME/{VRP-DATA-DIR}/graphhopper \
+-Dapp.routing.osm-dir=$HOME/{VRP-DATA-DIR}/openstreetmap \
+-Dapp.routing.osm-file=entral-america-latest.osm.pbf \
+-Dapp.region.country-codes=BZ,GT
```

Comment:
The `-jar quarkus-app/quarkus-run.jar` argument is missing in this command example. It should be at the last position after all `-D` arguments, so on line 85 (note that you'll need to add ` \` at the end of line 84).